src/mock-data/submitted-formr-partb.ts

```diff
@@ -228,3 +228,91 @@ export const submittedFormRPartBsWithUnsubmitted = [
     lastModifiedDate: "2020-04-16"
   }
 ];
+
+export const submittedFormRPartBwithCovid: FormRPartB[] = [
+  {
+    id: "5e972ec9b9b5781b94eb1270",
+    traineeTisId: "123",
+    forename: "Anthony Mara",
+    surname: "Gilliam",
+    gmcNumber: "11111111",
+    email: "email@email.com",
+    localOfficeName: "Health Education England Thames Valley",
+    prevRevalBody: "Health Education England Midlands",
+    prevRevalBodyOther: "",
+    prevRevalDate: "2020-04-22",
+    currRevalDate: "2020-04-22",
+    programmeSpecialty: "ST3",
+    dualSpecialty: "DS",
+    work: [
+      {
+        typeOfWork: "In Post ST1 Dermatology",
+        startDate: "2019-01-12",
+        endDate: "2019-12-21",
+        trainingPost: "Yes",
+        site: "Addenbrookes Hospital",
+        siteLocation: "Hills Road Cambridge Cambridgeshire"
+      },
+      {
+        typeOfWork: "In Post ST1 Dermatology",
+        startDate: "2020-01-01",
+        endDate: "2020-12-31",
+        trainingPost: "Yes",
+        site: "Addenbrookes Hospital",
+        siteLocation: "Hills Road Cambridge Cambridgeshire"
+      }
+    ],
+    sicknessAbsence: 0,
+    parentalLeave: 0,
+    careerBreaks: 0,
+    paidLeave: 0,
+    unauthorisedLeave: 10,
+    otherLeave: 0,
+    totalLeave: 10,
+    isHonest: true,
+    isHealthy: true,
+    isWarned: true,
+    isComplying: true,
+    healthStatement: "I feel great etc.",
+    havePreviousDeclarations: true,
+    havePreviousUnresolvedDeclarations: false,
+    previousDeclarations: [
+      {
+        declarationType: "Significant Event",
+        dateOfEntry: "2020-03-07",
+        title: "Previous declaration title",
+        locationOfEntry: "Previous declaration location of entry"
+      }
+    ],
+    previousDeclarationSummary: "",
+    haveCurrentDeclarations: true,
+    haveCurrentUnresolvedDeclarations: false,
+    currentDeclarations: [
+      {
+        declarationType: "Complaint",
+        dateOfEntry: "2020-06-12",
+        title: "Current declaration title",
+        locationOfEntry: "Current declaration location of entry"
+      }
+    ],
+    currentDeclarationSummary: "",
+    compliments: "",
+    haveCovidDeclarations: true,
+    covidDeclarationDto: {
```

Comment:
This content has been tailored to suit the test, which expects all the fields to be included in the output. As such, the test for Section 7 does not test any of the internal display logic of the component. I'm not sure if we should look at creating a few different test data records for this that allow us to validate the display logic, or if this is a bit OTT at this point...

Reply:
@ReubenRobertsHEE Sounds like a good idea - depends how much energy we've got after tacking the other stuff. Might be a one for another day? 😄

Reply:
Have just added some tests for the displaying logic in Section 7.